integration_tests/pom.xml

```diff
@@ -180,6 +180,36 @@
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>exec-maven-plugin</artifactId>
                 <executions>
+                    <execution>
+                        <id>clean previously populated config file</id>
```

Comment:
Although we have contradicting examples. Through out the diff: It will be easier to use if the `id` does not have spaces even if Maven allows it. Consider replacing spaces with `-`

Reply:
I did follow those contradicting examples even though I thought it's strange 🙂 We should fix those IDs that have spaces in them. I fixed the ones I added and another nearby.